jkube-kit/config/service/src/test/java/org/eclipse/jkube/kit/config/service/openshift/OpenshiftBuildServiceIntegrationTest.java

```diff
@@ -552,6 +562,7 @@ private WebServerEventCollector prepareMockServer(
         .build();
 
     if (!buildConfigExists) {
+      mockServer.expect().get().withPath("/api/v1/namespaces/ns1/pods?labelSelector=openshift.io%2Fbuild.name").andReply(collector.record("build-config-check").andReturn(200, bc)).always();
```

Comment:
You've changed the code to use build/logs, but why isn't that reflected in mock expectations? I see this expectation is for listing pods created for build.

Reply:
Yeah, you are right, however tests look for that resource - `pod`. Let me check it again and find a object which is using this API call

Reply:
@rohanKanojia hello
I found the issue: 
https://github.com/fabric8io/kubernetes-client/blob/c8163e53f7cb9344e8f6c85597a7eabe14189bc1/openshift-client/src/main/java/io/fabric8/openshift/client/dsl/internal/build/BuildOperationsImpl.java#L194

It is looking for `Pod` resource

Reply:
@arsenalzp : Thanks for clarification!

Reply:
Could this remark be resolved?